### PR TITLE
Add tests for UIImage.compressedData(quality:)

### PR DESCRIPTION
**Pull request by thisIsTheFoxe**
Added missing test for `UIImage.compressedData(quality:)`

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 5.0.
- [ ] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [x] I have added tests for [new] extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Tests/UIKitTests/UIImageExtensionsTests.swift [hunk 1]
```diff
@@ -47,6 +47,16 @@ final class UIImageExtensionsTests: XCTestCase {
         XCTAssertNil(UIImage().compressed())
     }
 
+        func testCompressedData() {
```
Comment (LucianoPAlmeida) [inline Tests/UIKitTests/UIImageExtensionsTests.swift]: Nit: Identation

File: CHANGELOG.md [hunk 2]
```diff
@@ -571,6 +571,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - `isInWeekday` has been renamed to `isWorkday`. [#313](https://github.com/SwifterSwift/SwifterSwift/pull/313) by [kaphacius](https://github.com/kaphacius).
 
 ### Enhancements
+- New **Image** test
```
Comment (LucianoPAlmeida) [inline CHANGELOG.md]: Here is not the place to add new entries, this should be added to the top on next version section ;)

**Comment by LucianoPAlmeida** [comment]
Nice, thank's for doing this :) Few minor comments

File: Tests/UIKitTests/UIImageExtensionsTests.swift [hunk 3]
```diff
@@ -47,6 +47,16 @@ final class UIImageExtensionsTests: XCTestCase {
...
+        let bundle = Bundle.init(for: UIImageExtensionsTests.self)
```
Comment (guykogus) [inline Tests/UIKitTests/UIImageExtensionsTests.swift]:
```suggestion
        let bundle = Bundle(for: UIImageExtensionsTests.self)
```

File: CHANGELOG.md (hunk 2 above)
Comment (thisIsTheFoxe) [reply CHANGELOG.md]: whoops.. .-.

File: Tests/UIKitTests/UIImageExtensionsTests.swift (hunk 1 above)
Comment (thisIsTheFoxe) [reply Tests/UIKitTests/UIImageExtensionsTests.swift]: ohh.. I thought SwiftLint would catch stuff like that.. 🤔

File: CHANGELOG.md [hunk 4]
```diff
@@ -22,6 +22,7 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **UIBarButtonItem**:
   - Added `flexibleSpace` and `fixedSpace(width:)` extensions to UIBarButtonItem to simplify the creation of spacers. [#728](https://github.com/SwifterSwift/SwifterSwift/pull/728) by [MaxHaertwig](https://github.com/maxhaertwig).
 - **UIImage**:
+  - Added missing test fot `compressedData(quality:)` by [@thisIsTheFoxe](https://github.com/thisisthefoxe) [#786]
```
Comment (guykogus) [inline CHANGELOG.md]:
```suggestion
  - Added missing test for `compressedData(quality:)` by [@thisIsTheFoxe](https://github.com/thisisthefoxe) [#786]
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -22,6 +22,7 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **UIBarButtonItem**:
   - Added `flexibleSpace` and `fixedSpace(width:)` extensions to UIBarButtonItem to simplify the creation of spacers. [#728](https://github.com/SwifterSwift/SwifterSwift/pull/728) by [MaxHaertwig](https://github.com/maxhaertwig).
 - **UIImage**:
+  - Added missing test for `compressedData(quality:)` by [@thisIsTheFoxe](https://github.com/thisisthefoxe) [#786]
```
Comment (guykogus) [inline CHANGELOG.md]:
```suggestion
  - Added missing test for `compressedData(quality:)`. [#786](https://github.com/SwifterSwift/SwifterSwift/pull/786) by [thisIsTheFoxe](https://github.com/thisisthefoxe)
```

File: Tests/UIKitTests/UIImageExtensionsTests.swift (hunk 3 above)
Comment (thisIsTheFoxe) [reply Tests/UIKitTests/UIImageExtensionsTests.swift]: Btw, if that's the preferred style, should I do another PR and just refactor the other Xyz.init(...)? .-.

Comment (guykogus) [reply Tests/UIKitTests/UIImageExtensionsTests.swift]: I wouldn't worry about it...

File: CHANGELOG.md (hunk 5 above)
Comment (thisIsTheFoxe) [reply CHANGELOG.md]: > There are no problems , only opportunities... xD

**Pull request merged**